tests/boardfarm_plugins/boardfarm_prplmesh/devices/prplmesh_docker.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, *args, **kwargs):
 
         docker_cmd = os.path.join(rootdir, "tools", "docker", "run.sh")
         docker_args = ["--verbose", "--detach", "--force", "--name", self.name,
-                       "--network", self.docker_network]
+                       "--network", self.docker_network, "--expose", "8002"]
```

Comment:
We should use the same approach as in test_gw_receiver.sh:
```
    # Read GW & Repeater UCC ports
    local GW_UCC_PORT
    GW_UCC_PORT=$(grep ucc_listener_port \
    "${rootdir}"/build/install/config/beerocks_controller.conf | \
    awk -F'[= ]' '{ print $2 }')

    local RP_UCC_PORT
    RP_UCC_PORT=$(grep ucc_listener_port \
    "${rootdir}"/build/install/config/beerocks_agent.conf | \
    awk -F'[= ]' '{ print $2 }')
```
and expose the correct one depending on the role.

Also, to improve it even more, for the controller, if the agent and controller UCC port are configured differently, we should expose both of them. That way we could do full tests with only a single device.